shared/types/api/node.go

```diff
@@ -265,11 +265,12 @@ type SendElRewardsResponse struct {
 }
 
 type CanWithdrawSdResponse struct {
-	Status                     string         `json:"status"`
-	Error                      string         `json:"error"`
-	InsufficientSdCollateral   bool           `json:"insufficientSdCollateral"`
-	InsufficientWithdrawableSd bool           `json:"insufficientWithdrawableSd"`
-	GasInfo                    stader.GasInfo `json:"gasInfo"`
+	Status                     string            `json:"status"`
+	Error                      string            `json:"error"`
+	InsufficientSdCollateral   bool              `json:"insufficientSdCollateral"`
+	InsufficientWithdrawableSd bool              `json:"insufficientWithdrawableSd"`
+	SdStatusResponse           *SdStatusResponse `json:"sdStatusResponse"`
```

Comment:
lets try to seperate this into a seperate get-sd-status cmd. It looks much cleaner like that according to me.